src/pages/workspace/WorkspacePageWithSections.js

```diff
@@ -65,6 +70,18 @@ const defaultProps = {
 
 class WorkspacePageWithSections extends React.Component {
     componentDidMount() {
+        this.fetchData();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (!prevProps.network.isOffline || this.props.network.isOffline) {
```

Comment:
I'm confused...

If we were previously offline, and now we are no longer online, wouldn't we want to `fetchData` instead of `return`?

Reply:
It is a little confusing but I think it's correct!
It is saying "If we were prev online (!prevPros.network.isOffline) or if we are offline right now, then returns"
Which means, if we just got offline/disconnected from the web don't do anything
But if it doesn't enter that if statement, then it just came back online.

This if statement comes from the parent issue:
https://github.com/Expensify/Expensify/issues/208185